sql/core/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -588,6 +588,12 @@ object SQLConf {
       .doubleConf
       .createWithDefault(0.05)
 
+  val IGNORE_CORRUPT_FILES = SQLConfigBuilder("spark.sql.files.ignoreCorruptFiles")
+    .doc("Whether to ignore corrupt files. If true, the Spark jobs will continue to run when " +
+      "encountering corrupt files and contents that have been read will still be returned.")
+    .booleanConf
+    .createWithDefault(false)
+
```

Comment:
Curious why we are duplicating the parameter in sql namespace. Wont spark.files.ignoreCorruptFiles not do ?